### PR TITLE
Refix i88!!!

### PR DESCRIPTION
**Pull request by hadialqattan**
@hakancelik96 I'm so sorry.

### CODE REVIEW

**Comment by hadialqattan** [comment]
Refix: #88 .

**Comment by isidentical** [comment]
Wait, why did you use an absolute http url instead of relatively linking the file `[BLABLA.MD](BLABLA.MD)`?

**Comment by hadialqattan** [comment]
> Wait, why did you use an absolute http url instead of relatively linking the file `[BLABLA.MD](BLABLA.MD)`?

This will fix the broken links in the CONTRIBUTING.md file(Github README) but introduce 404 Error in CHANGELOG.md and AUTHORS.md links the [Docs](https://unimport.hakancelik.dev/#/CONTRIBUTING?id=the-final-step).

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
 ## The final step
 
 After adding a new feature or fixing a bug please report your change to
-[CHANGELOG.md](/docs/CHANGELOG.md) and write your name, github address and email in the
-[AUTHORS.md](/docs/AUTHORS.md) file in alphabetical order.
+[CHANGELOG.md](https://github.com/hakancelik96/unimport/blob/master/docs/CHANGELOG.md)
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:61]:
```suggestion
[CHANGELOG.md](CHANGELOG.md)
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
...
+and write your name, github address and email in the
+[AUTHORS.md](https://github.com/hakancelik96/unimport/blob/master/docs/AUTHORS.md) file
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:63]:
```suggestion
[AUTHORS.md](AUTHORS.md)
```

**Review by hakancelik96 — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
 ## The final step
 
 After adding a new feature or fixing a bug please report your change to
-[CHANGELOG.md](/docs/CHANGELOG.md) and write your name, github address and email in the
-[AUTHORS.md](/docs/AUTHORS.md) file in alphabetical order.
+[CHANGELOG.md](CHANGELOG.md)
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:61]:
```suggestion
[CHANGELOG.md](CHANGELOG.md) and write your name, GitHub address, and email in the
```

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
...
+and write your name, github address and email in the
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:62]:
```suggestion
```

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
...
+[AUTHORS.md](AUTHORS.md) file
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:63]:
```suggestion
[AUTHORS.md](AUTHORS.md) file in alphabetical order.
```

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -58,8 +58,10 @@ If all tests pass.
...
+in alphabetical order.
```
Comment (hakancelik96) [inline docs/CONTRIBUTING.md:64]:
```suggestion
```

**Review by hakancelik96 — CHANGES_REQUESTED** [review]

**Pull request merged**